lib/services.js

```diff
@@ -450,6 +450,7 @@ export default {
             metadata.putObjectMD(bucketName, longMPUIdentifier,
                 multipartObjectMD, log, (err) => {
                     if (err) {
+                        log.error('error from metadata', { error: err });
```

Comment:
The reason nothing was logged here is due to this log entry: https://github.com/scality/S3/blob/master/lib/metadata/wrapper.js#L74

Maybe bumping it to "error" instead of "warn" would be enough ?